Sources/PointFree/Account/Team.swift

```diff
@@ -116,7 +113,7 @@ private func sendEmailsForTeammateRemoval(owner: User, teammate: User) -> Parall
     parallel(sendEmail(
       to: [teammate.email],
       subject: "You have been removed from \(owner.displayName)’s Point-Free team",
-      content: inj2(youHaveBeenRemovedEmailView.view((owner, teammate)))
+      content: inj2(youHaveBeenRemovedEmailView.view(.teamOwner(owner)))
```

Comment:
I generalized the `youHaveBeenRemovedEmailView` to be usable for both team removals and enterprise removals.